pkg/agent/openflow/client.go

```diff
@@ -113,13 +114,6 @@ type Client interface {
 	InstallServiceFlows(groupID binding.GroupIDType, svcIP net.IP, svcPort uint16, protocol binding.Protocol, affinityTimeout uint16, nodeLocalExternal bool, svcType v1.ServiceType) error
 	// UninstallServiceFlows removes flows installed by InstallServiceFlows.
 	UninstallServiceFlows(svcIP net.IP, svcPort uint16, protocol binding.Protocol) error
-	// InstallLoadBalancerServiceFromOutsideFlows installs flows for LoadBalancer Service traffic from outside node.
-	// The traffic is received from uplink port and will be forwarded to gateway by the installed flows. And then
-	// kube-proxy will handle the traffic.
-	// This function is only used for Windows platform.
-	InstallLoadBalancerServiceFromOutsideFlows(svcIP net.IP, svcPort uint16, protocol binding.Protocol) error
```

Comment:
Since we remove function `InstallLoadBalancerServiceFromOutsideFlows` here, I think we need to enable IP Forwarding on OVS bridge interface to ensure the outside LoadBalancer traffic can be forwarded to kube-proxy from br-int. Could you also add this logic？

Reply:
I think another PR #3137 can do this.

Reply:
In fact, current main branch code will lead to broken function as UplinkTable is bypassed. We need PR #3137 to fix the issue.